.travis/run.sh

```diff
@@ -10,4 +10,4 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     pyenv activate conan
 fi
 
-python build.py
\ No newline at end of file
+conan create . travis/testing -s compiler=clang -s compiler.version=5.0 -s compiler.libcxx=libstdc++ -e CXX=clang++
```

Comment:
Removing the conan-package-tools will remove the automatic upload to bintray?

Reply:
Oops, I didn't know there was such a feature :/

Should I revert the removal of `build.py`?

Reply:
Yep :)

I think so, it is very convenient that the upload is automatic, in that way @vthiery just have to merge contributions and packages will be automatically uploaded, no need to do it manually.

It can also be done with a ``conan upload`` in the scripts but you need first to set the remote, then the user credentials for that remote... maybe better revert build.py, yes

Reply:
Ok, my bad. Will do

Reply:
Ouch missed that! Yes indeed, the automatic upload is quite convenient, would be nice to keep it around :)